bitnami/kubeapps/values.yaml

```diff
@@ -144,7 +144,7 @@ frontend:
   image:
     registry: docker.io
     repository: bitnami/nginx
-    tag: 1.21.1-debian-10-r1
+    tag: 1.20.1-debian-10-r32
```

Comment:
The current (more recently updated) version should be kept. Same for oauth2-proxy and the other nginx entry below. We should only see pinniped-proxy being updated to 2.3.3 here.